svelte-motoko-starter/src/frontend/rollup.config.js

```diff
@@ -0,0 +1,156 @@
+import svelte from "rollup-plugin-svelte";
+import commonjs from "@rollup/plugin-commonjs";
+import resolve from "@rollup/plugin-node-resolve";
+import livereload from "rollup-plugin-livereload";
+import { terser } from "rollup-plugin-terser";
+import css from "rollup-plugin-css-only";
+import replace from "@rollup/plugin-replace";
+import inject from "rollup-plugin-inject";
+import json from "@rollup/plugin-json";
+
+const production = !process.env.ROLLUP_WATCH;
+
+const path = require("path");
+
+function initCanisterIds() {
```

Comment:
There's a newer version of this in the starter webpack config, that may be a little simpler to integrate with rollup here: https://github.com/dfinity/sdk/blob/master/src/dfx/assets/new_project_node_files/webpack.config.js#L7

The new version creates a single map of all the canister ids to their capitalCase environment variable name, and requires less manual overriding from devs